.github/workflows/main.yml

```diff
@@ -0,0 +1,289 @@
+# --------------------------------------------
+# This is a basic workflow to help you get started with Actions
+# --------------------------------------------
+name: Build
+
+on:
+  # --------------------------------------------
+  # Controls when the automatic build job will be run.
+  # By default uses the default branch. Runs at 03:00 UTC.
```

Comment:
Outdated comment? It runs at 05:00 UTC, right? Maybe we could remove that sentence completely...